services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:

```

Comment:
If argument can be None, you must show it in typehint. Fix everywhere, please
```suggestion
def get_movies(genres_ids: list[int] | None = None,
               actors_ids: list[int] | None = None) -> QuerySet:

```